config-eslint/typescript.js

```diff
@@ -80,6 +80,7 @@ module.exports = {
           'error',
           { assertionStyle: 'as', objectLiteralTypeAssertions: 'allow-as-parameter' },
         ],
+        '@typescript-eslint/consistent-type-imports': ['error', { prefer: 'type-imports' }],
```

Comment:
enabled auto-fixing `import { FooType } from './foo'` to `import type { FooType } from './foo'`

Reply:
Typescript should not require this afaik unless `verbatimModuleSyntax` is enabled 🤔

But it's a good practice anyway, so 👍

Reply:
I think I was getting `*.d.ts is not a module` errors, but actually fixed that in tsconfig with `"moduleResolution": "node16"`

but agree this is best practice/we should do it anyway 👍